File: Python-Task-02/C/password_checker.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main() {
    char password[100];
    int hasUpper = 0;
    int hasDigit = 0;

    printf("Enter password: ");
    scanf("%s", password);

    int length = strlen(password);

    for (int i = 0; i < length; i++) {
        if (isupper(password[i]))
            hasUpper = 1;

        if (isdigit(password[i]))
            hasDigit = 1;
    }

    if (length >= 8 && hasUpper && hasDigit)
        printf("Strong Password\n");
    else if (length >= 8 && (hasUpper || hasDigit))
        printf("Moderate Password\n");
    else
        printf("Weak Password\n");

    return 0;
}

File: Python-Task-02/python/password_checker.py
password = input("Enter password: ")

has_upper = False
has_digit = False

for char in password:
    if char.isupper():
        has_upper = True

    if char.isdigit():
        has_digit = True

if len(password) >= 8 and has_upper and has_digit:
    print("Strong Password")
elif len(password) >= 8 and (has_upper or has_digit):
    print("Moderate Password")
else:
    print("Weak Password")